Content.Shared/Chat/Prototypes/EmoteSoundsPrototype.cs

```diff
@@ -2,36 +2,49 @@
 using Robust.Shared.Prototypes;
 using Robust.Shared.Serialization;
 using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Dictionary;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Array;
 
 namespace Content.Shared.Chat.Prototypes;
 
 /// <summary>
 ///     Sounds collection for each <see cref="EmotePrototype"/>.
 ///     Different entities may use different sounds collections.
 /// </summary>
-[Prototype]
-public sealed partial class EmoteSoundsPrototype : IPrototype
+[Prototype, Serializable, NetSerializable]
```

Comment:
```suggestion
[Prototype]
```
you shouldn't be serializing prototypes. Simply send over the ID on the network and index on the client.